### DISCUSSION TITLE
2007/01/16/%E4%B8%8D%E8%A6%81%E6%80%95%EF%BC%8C%E4%B8%8D%E8%A6%81%E6%82%94

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#139] [Announcements]
# 2007/01/16/%E4%B8%8D%E8%A6%81%E6%80%95%EF%BC%8C%E4%B8%8D%E8%A6%81%E6%82%94

今天老妈突然打电话来跟我讨论了一下股票，说最近股市形势很好啊。我老家那里算是祖国边疆了，连从来不接触股票的老妈都听说股市不错，可见过去一段时间的股市大涨影响之广，差不多能吸引进股市的人都已经被吸进去了吧？不过按照以往的规律，一般到了这个时候股市也就危险了。

https://blog.qizhen.xyz/2007/01/16/%E4%B8%8D%E8%A6%81%E6%80%95%EF%BC%8C%E4%B8%8D%E8%A6%81%E6%82%94

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yoyo
2007年01月19日 17:52 编辑
哥儿们，您还是适合买基金

回复
YY
2007年01月18日 23:35 编辑
哈哈，把买股票当成企业来买我想就比较好了，重要的不是股价，是企业的业绩和前途：）
